opentech/apply/funds/api_views.py

```diff
@@ -85,7 +85,16 @@ def post(self, request, *args, **kwargs):
             obj.perform_transition(action, self.request.user, request=self.request)
```

Comment:
Missing an redirect check here.

there should be two checks.
1. Should this have been a redirect? - if so they have stale data or trying to break things
  * we should prevent that
  * determinations are not coded into the workflow explicitly. so its a seperate step.
2. Do the transition (and report if it fails) - as it currently does

Reply:
Updated.